lt/client/client.go

```diff
@@ -468,6 +468,10 @@ func (u *User) Speak(text string) chan struct{} {
 }
 
 func (u *User) onConnect() {
+	// Waiting a second before transmitting to avoid
+	// the known race between incoming and outgoing track.
+	// This is needed as pion doesn't support perfect negotiation yet.
+	time.Sleep(time.Second)
```

Comment:
Using DC further exacerbated the impact of signaling conflicts, so I had to add this to make it usable. Browsers are not affected as they support perfect negotiation (SDP rollback).

Reply:
Gotcha. :+1: